VKApp/VKApp/Model/Group/GroupResponse.swift

```diff
@@ -4,7 +4,9 @@
 import RealmSwift
 
 /// GroupResponse
```

Comment:
```suggestion
/// GroupResponse
```
Или все коменты на русском или все на английском